File: tests/fixtures/ruby/bumpy_road.rb
def bumpy_process(items)
  result = 0
  if items.length > 0
    if items[0] > 0
      if items[0] > 10
        result += 1
      end
    end
  end
  result += 1
  if items.length > 5
    if items[5] > 0
      if items[5] > 10
        result += 2
      end
    end
  end
  result += 2
  if items.length > 10
    if items[10] > 0
      if items[10] > 10
        result += 3
      end
    end
  end
  result
end

File: tests/fixtures/zig/bumpy_road.zig
const std = @import("std");

pub fn bumpyProcess(items: []const u8) i32 {
    var result: i32 = 0;
    if (items.len > 0) {
        if (items[0] > 0) {
            if (items[0] > 10) {
                result += 1;
            }
        }
    }
    result += 1;
    if (items.len > 5) {
        if (items[5] > 0) {
            if (items[5] > 10) {
                result += 2;
            }
        }
    }
    result += 2;
    if (items.len > 10) {
        if (items[10] > 0) {
            if (items[10] > 10) {
                result += 3;
            }
        }
    }
    return result;
}
